fe/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -643,16 +634,12 @@ public static void checkAndCreateSource(Database db, DataDescription dataDescrip
                 throw new DdlException("Load for AGG_KEYS table should not specify NEGATIVE");
             }
 
-            if (((OlapTable) table).getKeysType() != KeysType.UNIQUE_KEYS && deleteFlag) {
-                throw new DdlException("Delete flag can only be used for UNIQUE_KEYS table");
-            }
-
             // get table schema
-            List<Column> tableSchema = table.getBaseSchema();
-            Map<String, Column> nameToTableColumn = Maps.newTreeMap(String.CASE_INSENSITIVE_ORDER);
-            for (Column column : tableSchema) {
-                nameToTableColumn.put(column.getName(), column);
-            }
+            List<Column> baseSchema = table.getBaseSchema();
+            // fill the column info if user does not specify them
+            dataDescription.fillColumnInfoIfNotSpecified(baseSchema);
```

Comment:
The `columns` and `parsedColumnExprList` include the same columns.

Reply:
No, In `fillColumnInfoIfNotSpecified()`, I just want to fill the fields which user did not fill. After that, these fields should look like user fill them, not some "after-analyzed" results.